README.md

```diff
@@ -56,6 +56,25 @@ The **arches_her/install/requirements.txt** file will also need to be edited in
 
 
 
+## Running Arches in a Docker Development Environment
+
+You can also run Arches in a Docker development environment. To do this, pull the `arches` repo and use the two compose files `docker-compose-dependencies.yml` and `docker-compose.yml`.
+
+- Clone the [`arches`](https://github.com/archesproject/arches.git) repository
```

Comment:
Needs to add at the end that the Arches repo needs to be cloned into the same root folder director as the arches_her repo.

We also need to let the user know that the `dev/7.5.x` branch (or the same development branch as listed in the `arches_her/arches_her/install/requirements.txt`) for arches is checked out before the compose up stage.

Reply:
Done